pkg/handler/v2/application_installation/application_installation.go

```diff
@@ -184,11 +204,6 @@ func convertExternalToInternal(app *apiv2.ApplicationInstallation) *appskubermat
 			Namespace: app.Namespace,
 		},
 		Spec: *app.Spec,
-		Status: appskubermaticv1.ApplicationInstallationStatus{
```

Comment:
status is only set by the controller and will be ignored by create/update operation. So it makes no sense to set it.

IMHO we should remove the status from `createApplicationInstallationReq` / `updateApplicationInstallationReq`

@SimonTheLeg WDYT

Reply:
there already is an issue for that #0365